community/cypher/cypher-compiler-3.1/src/main/scala/org/neo4j/cypher/internal/compiler/v3_1/ast/rewriters/nameMatchAndMergePatternElements.scala

```diff
@@ -22,14 +22,17 @@ package org.neo4j.cypher.internal.compiler.v3_1.ast.rewriters
 import org.neo4j.cypher.internal.frontend.v3_1.ast._
 import org.neo4j.cypher.internal.frontend.v3_1.{Rewriter, bottomUp}
 
-case object nameMatchPatternElements extends Rewriter {
+case object nameMatchAndMergePatternElements extends Rewriter {
```

Comment:
There is already a nameUpdatingClause Rewriter that does both Match and Merge. I think we can get rid of this one entirely.